doc/changes/changes_1.2.0.md

```diff
@@ -1,4 +1,4 @@
-# Spark Exasol Connector 1.2.0, released 2021-12-14
+# Spark Exasol Connector 1.2.0, released 2021-12-??
```

Comment:
will the released date be filled by the release-droid?

Reply:
Forgot this, actually wanted to update it to today. Nice catch, thanks!